policies/policy-change-process.md

```diff
@@ -0,0 +1,59 @@
+Policy change process policy
+============================
+
+This policy represents the way how any additions or changes to the existing
+policies are proposed, edited, finalized, and approved.
+
+Policy change proposal
+----------------------
+
+The policy changes or additions are submitted as pull requests in the
+technical-policies repository on GitHub OpenSSL project.
+
+Each policy is placed in an individual file in the MarkDown format in the
+policies subdirectory.
+
+Any policy change proposal can modify any number of policies as required.
+
+Any policy change proposal SHOULD have a single topic.
+
+The description of the pull request SHOULD provide overview of the changes
+and the reasons why the change is proposed.
+
+After the pull request is created the author SHOULD announce the policy
+change proposal on the openssl-project mailing list.
```

Comment:
I think a MUST is better here, since the two weeks deadline below also is a MUST.

```suggestion
After the pull request is created the author MUST announce the policy
change proposal on the openssl-project mailing list.
```

_(Alternatively, the opening of pull requests could be posted by some GitHub action or bot?)_